cwms-data-api/src/test/resources/cwms/cda/api/gate_change.json

```diff
@@ -0,0 +1,39 @@
+[ {
+  "type" : "gate-change",
+  "project-id" : {
+    "office-id" : "SPK",
+    "name" : "PROJECT1"
```

Comment:
use a "real" project name.

Reply:
Swapped for a project name used in other Outlet testing